stable/mongodb/values.schema.json

```diff
@@ -0,0 +1,141 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "usePassword": {
+      "type": "boolean",
+      "title": "Enable password authentication",
+      "form": true
+    },
+    "mongodbRootPassword": {
+      "type": "string",
+      "title": "MongoDB admin password",
+      "form": true,
+      "description": "Defaults to a random 10-character alphanumeric string if not set",
+      "hidden": {
+        "condition": false,
+        "value": "usePassword"
+      }
+    },
+    "mongodbUsername": {
+      "type": "string",
+      "title": "MongoDB custom user",
+      "form": true
+    },
+    "mongodbDatabase": {
+      "type": "string",
+      "title": "MongoDB custom database",
+      "form": true
+    },
+    "mongodbPassword": {
+      "type": "string",
+      "title": "Password for MongoDB custom user",
+      "form": true,
+      "description": "Defaults to a random 10-character alphanumeric string if not set",
+      "hidden": {
+        "condition": false,
+        "value": "usePassword"
+      }
+    },
+    "replicaSet": {
+      "type": "object",
+      "title": "Replicaset configuration",
+      "form": true,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Enable replicaset configuration"
+        },
+        "replicas": {
+          "type": "object",
+          "title": "Number of replicas",
+          "form": true,
+          "hidden": {
+            "condition": false,
+            "value": "replicaSet.enabled"
+          },
+          "properties": {
+            "secondary": {
+              "type": "integer",
+              "title": "Secondary node replicas",
+              "description": "Number of secondary node replicas to deploy",
+              "form": true
+            },
+            "arbiter": {
+              "type": "integer",
+              "title": "Arbiter node replicas",
+              "description": "Number of arbiter node replicas to deploy",
+              "form": true
+            }
+          }
+        }
+      }
+    },
+    "persistence": {
+      "type": "object",
+      "title": "Persistence configuration",
+      "form": true,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Enable persistence",
+          "description": "Enable persistence using Persistent Volume Claims"
+        },
+        "size": {
+          "type": "string",
+          "title": "Persistent Volume Size",
+          "form": true,
+          "render": "slider",
+          "sliderMin": 1,
+          "sliderMax": 100,
+          "sliderUnit": "Gi",
+          "hidden": {
+            "condition": false,
+            "value": "persistence.enabled"
+          }
+        }
+      }
+    },
+    "volumePermissions": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
```

Comment:
should you hide this param if `persistence.enabled` is false?

Reply:
Yes, we should.